### DISCUSSION TITLE
Variants with unknown breakpoints and only one known affected position

### DISCUSSION THREAD

**Discussion by ifokkema** [HGVSnomenclature/hgvs-nomenclature#204] [Q&A]
While going over some special cases in my list of weird variants, I came across this one: [`NC_000016.9:g.(?_2097990)_?del`](https://databases.lovd.nl/shared/variants/0000633070#00021926). A similar, but in a way reversed, notation found was [`NC_000003.11:g.?_(69915421_?)dup`](https://databases.lovd.nl/shared/variants/0000698965#00013652).

When only a single position is reported as affected (`A`) and the breakpoints are unknown, whether to use `g.?_(A_?)` or `g.(?_A)_?` seems to be an arbitrary choice. Whether having only one known affected position can be prevented or not depends on the exact method used to find this variant, but I feel that the HGVS nomenclature could discourage the use of this arbitrary choice and recommend `g.(?_A)_(A_?)` instead. I.e., the variants would then become `NC_000016.9:g.(?_2097990)_(2097990_?)del` and `NC_000003.11:g.(?_69915421)_(69915421_?)dup`, respectively.

**Comment by jtdendunnen** [comment]
Good catch, these should indeed be corrected as you indicated, like NC_000016.9:g.(?_2097990)_(2097990_?)del.
